src/vrpn_client_ros.cpp

```diff
@@ -222,143 +223,206 @@ namespace vrpn_client_ros
 
       tf_broadcaster.sendTransform(tracker->transform_stamped_);
     }
+
+
+    if (tracker->calculate_twist_and_accel_ && tracker->previous_message_arrived_)
```

Comment:
This is an interesting feature! Honestly I think it probably belongs as another node in the system, consuming 'pose' messages and spitting out 'twist' and/or 'accel'. that keeps the VRPN client 'simpler' in that it's responsible only for adapting the incoming data.

Reply:
Agree completely. I will remove that func from this package and some more commits to PR

Reply:
Removed in a9db176